packages/http-server/src/server.ts

```diff
@@ -20,9 +18,9 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
     logger: (components && components.logger) || createLogger('HTTP SERVER'),
     disableRequestLogging: true,
     modifyCoreObjects: false,
-  })
-    .register(formbodyParser)
-    .register(fastifyAcceptsSerializer, { serializers });
+  }).register(formbodyParser);
+
+  server.setReplySerializer(payload => payload as string);
```

Comment:
This line can be removed if we set the serialiser in the response. See the other comment for other detail.

Reply:
Done 👍